certified-connectors/Emigo/apiDefinition.swagger.json

```diff
@@ -1,9 +1,9 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "1.0",
+    "version": "1.2",
     "title": "Emigo",
-    "description": "It enables the achievement of goals set for the sales force. It ensures effective activities in shops, pharmacies or with medical staff. Increases the effectiveness of sales teams in their daily activities. Increases efficiency.",
+    "description": "Emigo enables effective management of field force teams (employees working in the field on smartphones or tablets). Increases their effectiveness in their daily activities. Emigo Connector interacts with Power Automate and Power Apps services. It enables business & back-office users to access data from Emigo to automate data flows & streamline business processes and digitalization of workflows happening outside of Emigo system.",
```

Comment:
@tomaszsagra The value 'description' contains at least one of the restricted words: 'power apps, power automate, logic apps'. Please see our docs [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
Hi, I uploaded a fix